_src/static/sass/main.scss

```diff
@@ -0,0 +1,66 @@
+// ------------------------------
+// edX [application-name]: Main Style Compile
+
+// About: Sass compile for the edX [application-name]. This does not contain styles for other edX products/experiences (e.g. account/onboarding).
+
+// ------------------------------
+// #VENDOR
+// ------------------------------
+@import '../../vendor/bourbon/bourbon';
+@import '../../vendor/neat/neat';
+// @import '../../vendor/jquery-ui';
+@import '../../vendor/bi-app/bi-app-ltr';
+
+// ------------------------------
+// #UTILITIES
+// ------------------------------
+@import 'base/variables';
+@import 'utilities/functions';
+@import 'utilities/mixins';
+@import 'utilities/helpers';
+/**/
+// @import 'themes/[theme]/utilities/utilities.scss';
+
+
+// ------------------------------
+// #BASE
+// ------------------------------
+@import 'base/normalize';
+@import 'base/base';
+@import 'base/typography';
+
+
+// ------------------------------
```

Comment:
If we're not using these partials now, my recommendation would be to remove it. I'd prefer to add them in as we add in new components/patterns per story.